homeassistant/components/shelly/sensor.py

```diff
@@ -1270,7 +1270,7 @@ def __init__(
         entity_category=EntityCategory.DIAGNOSTIC,
         use_polling_coordinator=True,
     ),
-    "humidity_0": RpcSensorDescription(
+    "humidity_rh": RpcSensorDescription(
```

Comment:
I understand the need to tidy up old mistakes, but we can't rename this as a "code quality PR" since it breaks the unique id of the devices and especially for battery operated devices there is not even a way to recover from it since we need to restore the value at startup.
<img width="568" height="193" alt="image" src="https://github.com/user-attachments/assets/a4d6f558-5b76-4af0-b2e7-be70f7dfd3f9" />

If we want to go with this path we need to migrate existing entities